tests/test_incomprrandmeth.py

```diff
@@ -31,47 +31,47 @@ def setUp(self):
         self.assertAlmostEqual(modes[0, 0], 1.49469700)
         self.assertAlmostEqual(modes[0, 1], 1.38687858)
         self.assertAlmostEqual(modes[1, 0], -0.27245271)
 
-    def test_struct_2d(self):
-        modes = self.rm_2d(self.x_grid, self.y_grid, mesh_type="structured")
-        self.assertAlmostEqual(modes[0, 0, 0], 0.50751115)
-        self.assertAlmostEqual(modes[0, 1, 0], 0.69751927)
-        self.assertAlmostEqual(modes[1, 1, 1], -0.19747468)
+    # def test_struct_2d(self):
+    #     modes = self.rm_2d(self.x_grid, self.y_grid, mesh_type="structured")
+    #     self.assertAlmostEqual(modes[0, 0, 0], 0.50751115)
+    #     self.assertAlmostEqual(modes[0, 1, 0], 0.69751927)
+    #     self.assertAlmostEqual(modes[1, 1, 1], -0.19747468)
 
-    def test_struct_3d(self):
-        modes = self.rm_3d(
-            self.x_grid, self.y_grid, self.z_grid, mesh_type="structured"
-        )
-        self.assertAlmostEqual(modes[0, 0, 0, 0], 1.49469700)
-        self.assertAlmostEqual(modes[1, 0, 1, 1], 0.12813365)
-        self.assertAlmostEqual(modes[1, 1, 0, 1], 0.01443056)
-        self.assertAlmostEqual(modes[1, 1, 1, 1], -0.12304040)
+    # def test_struct_3d(self):
+    #     modes = self.rm_3d(
+    #         self.x_grid, self.y_grid, self.z_grid, mesh_type="structured"
+    #     )
+    #     self.assertAlmostEqual(modes[0, 0, 0, 0], 1.49469700)
+    #     self.assertAlmostEqual(modes[1, 0, 1, 1], 0.12813365)
+    #     self.assertAlmostEqual(modes[1, 1, 0, 1], 0.01443056)
+    #     self.assertAlmostEqual(modes[1, 1, 1, 1], -0.12304040)
 
-    def test_struct_unstruct(self):
-        x_grid = np.arange(0.0, 2.0, 1.0)
-        y_grid = np.arange(0.0, 2.0, 1.0)
-        x_tuple = np.array((0.0, 0.0, 1.0, 1.0))
-        y_tuple = np.array((0.0, 1.0, 0.0, 1.0))
-        unstr_modes = self.rm_2d(x_tuple, y_tuple, mesh_type="unstructured")
-        str_modes = self.rm_2d(x_grid, y_grid, mesh_type="structured")
-        for d in range(2):
-            k = 0
-            for i in range(len(x_grid)):
-                for j in range(len(y_grid)):
-                    self.assertAlmostEqual(
-                        str_modes[d, i, j], unstr_modes[d, k]
-                    )
-                    k += 1
+    # def test_struct_unstruct(self):
+    #     x_grid = np.arange(0.0, 2.0, 1.0)
+    #     y_grid = np.arange(0.0, 2.0, 1.0)
+    #     x_tuple = np.array((0.0, 0.0, 1.0, 1.0))
+    #     y_tuple = np.array((0.0, 1.0, 0.0, 1.0))
+    #     unstr_modes = self.rm_2d(x_tuple, y_tuple, mesh_type="unstructured")
+    #     str_modes = self.rm_2d(x_grid, y_grid, mesh_type="structured")
+    #     for d in range(2):
+    #         k = 0
+    #         for i in range(len(x_grid)):
+    #             for j in range(len(y_grid)):
+    #                 self.assertAlmostEqual(
+    #                     str_modes[d, i, j], unstr_modes[d, k]
+    #                 )
+    #                 k += 1
```

Comment:
Are you planing on putting similar tests to `test_srf.py`?

Reply:
Since this (low-level) functionality is not present anymore, we don't need to test it anymore.